fun/module.py

```diff
@@ -63,7 +63,12 @@ async def highfive(self, ctx, *, user: nextcord.Member):
 
         The user has to highfive you in under twenty seconds.
         """
-        if user is not None and user.id not in [m.id for m in ctx.channel.members]:
+        if isinstance(ctx.channel, nextcord.Thread):
+            # in Threads attribute members is not set correctly
+            members =  await ctx.channel.fetch_members()
```

Comment:
Flake8 correctly flagged this as an issue, you have two spaces there. Otherwise it's fine, will merge when this gets fixed.